queries/idris/highlights.scm

```diff
@@ -0,0 +1,213 @@
+; ------------------------------------------------------------------------------
+; Literals and comments
+[
+  (integer)
+  (quantity)
+] @number
+
+(literal
+  (number)) @number.float
+
+(char) @character
+
+[
+  (string)
+  (pat_string)
+  (triple_quote_string)
+] @string.documentation
+
+(comment) @comment
+
+; more general captures are moved to the top
+; before overwritten later by more specific captures
+(caname) @constructor
```

Comment:
Is there a reason for keeping this? Seems it already applies to most cases below (e.g. in `exp_name`, etc.). Also this overrides `module foo`.